.circleci/config.yml

```diff
@@ -17,6 +17,25 @@ jobs:
             OPS_DIR=$HOME/.ops
             PATH=$HOME/.ops/bin:$PATH
             make test-noaccel
+      - run:
+          command: |
+            if [[ ! -v GCLOUD_SERVICE_KEY ]]; then
+              circleci-agent step halt
+            fi
+      - run: echo "deb http://packages.cloud.google.com/apt cloud-sdk main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+      - run: curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+      - run: sudo apt-get update && sudo apt-get install google-cloud-sdk
+      - run: echo $GCLOUD_SERVICE_KEY | gcloud auth activate-service-account --key-file=-
+      - run: gcloud config set project ${GOOGLE_PROJECT_ID}
+      - run: gcloud --quiet config set compute/zone ${GOOGLE_COMPUTE_ZONE}
+      - run: mkdir temp && cp output/platform/pc/bin/kernel.img temp/ && cp output/platform/pc/boot/boot.img temp/ && cp output/platform/pc/boot/bootx64.efi temp/ && mkdir temp/klibs && cp output/klib/bin/* temp/klibs
+      - run: echo "export COMMIT_ID="`git rev-parse --short=7 HEAD` >> $BASH_ENV
+      - run: cd temp && tar cvzf ../nanos-release-linux-${COMMIT_ID}.tar.gz *
+      - run: gsutil cp nanos-release-linux-${COMMIT_ID}.tar.gz gs://nanos/release/${COMMIT_ID}/
+      - run: mv nanos-release-linux-${COMMIT_ID}.tar.gz nanos-release-darwin-${COMMIT_ID}.tar.gz
```

Comment:
Is it possible to do ```gsutil cp gs://nanos/release/${COMMIT_ID}/nanos-release-linux-${COMMIT_ID}.tar.gz gs://nanos/release/${COMMIT_ID}/nanos-release-darwin-${COMMIT_ID}.tar.gz``` instead of locally copying and uploading the same image?

Reply:
Ye, indeed it's possible to copy files within a given bucket, I changed the commands so that after uploading the linux file, this file is copied directly to the darwin file within the bucket.